File: UI/newmark.property.app/src/components/Property.js
import React from "react";
import Space from "./Space";

const Property = ({ property }) => {
  return (
    <div className="property">
      <h2>{property.name}</h2>
      <p><strong>Features:</strong> {property.features.join(", ")}</p>
      <p><strong>Highlights:</strong> {property.highlights.join(", ")}</p>
      <p><strong>Transportation:</strong> {property.transportation.join(", ")}</p>

      <div className="spaces">
        <h3>Spaces:</h3>
        {property.spaces.map((space, index) => (
          <Space key={index} space={space} />
        ))}
      </div>
    </div>
  );
};

export default Property;

File: API/NewMark.PropertyManagement.Models/Property.cs
namespace NewMark.PropertyManagement.Models
{

    /// <summary>
    /// Represents a property with the necessary details.
    /// </summary>
    public class Property
    {
        public string PropertyId { get; set; }
        public string PropertyName { get; set; }
        public List<string> Features { get; set; } = new List<string>();
        public List<string> Highlights { get; set; } = new List<string>();
        public List<Transportation> Transportation { get; set; } = new List<Transportation>();
        public List<Space> Spaces { get; set; } = new List<Space>();
    }
}